markdown/org/docs/patterns/lunetius/cutting/en.md

```diff
@@ -1,3 +1,21 @@
-Lunetius consists only of one part, the *lacerna* itself:
+---
+title: Cutting Lunetius
+---
+
+The way to cut Lunetius is going to depend on what fabric you have chosen to use. If you have enough fabric width you can cut it on the fold. If you don’t you can cut two halves and later sew them together. To not have to later finish the centre back edge you can cut with the centre back edge on the selvage.
+
+A lot of garments were woven to shape in the roman era, so this is one aspect where you don’t need to worry about historical accuracy at all, unless you would like to weave your Lunetius to shape. <Comment by="Zee">In that case I salute you and please share the results in the discord!</Comment>
```

Comment:
Please start a new paragraph when using the `Comment` (or any) custom component.
This ensures the parser picks up on it.

Reply:
Done this too!